app/requestService.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+class RequestClient {
+  constructor() {
+    this.request = require('request');
```

Comment:
Poza tym dobrze jakbyś nie korzystał z klas w node. Nie jest to wgle potrzebne. Najlepiej jest deklarować wszystkie funkcjonalności jako named functions (arrow if no "this" needed) i następnie eksportować je na końcu pliku jako np.:
```javascript
module.export = {
  funcToExport,
};
```
Lub w jakikolwiek inny sposób (bylebyś trzymał się go w całym projekcie